.java-version

```diff
@@ -0,0 +1 @@
+1.8
```

Comment:
Removed, was probably added because I had some problems running e2e tests locally and thought it might be a java version issue (which was changed to 11 globally so have java environment files to test with other versions)